File: 20190915-csharp-dependency-breaking-katas/DependencyBreakingKatas/B/MarketingCampaignTest.cs
using Xunit;

namespace Dependencies.B
{
    public class MarketingCampaignTest
    {
        [Fact]
        public void Test2()
        {
            var campaign = new MarketingCampaign();

            var isCrazySalesDay = campaign.IsCrazySalesDay();

            Assert.False(isCrazySalesDay);
        }
    }
}

File: 20190923-java-dependency-breaking-katas/src/test/java/org/codecop/dependencies/b/MarketingCampaignTest.java
package org.codecop.dependencies.b;

import org.junit.Test;

import static junit.framework.TestCase.assertTrue;

public class MarketingCampaignTest {

    @Test
    public void should_be_crazy_sales_day_on_tuesdays() {
        MarketingCampaign campaign = new MarketingCampaign();

        boolean isCrazySalesDay = campaign.isCrazySalesDay();

        assertTrue(isCrazySalesDay);
    }
}
